src/main/flight/autopilot.c

```diff
@@ -53,131 +54,138 @@ static pidCoefficient_t positionPidCoeffs;
 static float altitudeI = 0.0f;
 static float throttleOut = 0.0f;
 
-typedef struct earthFrame_s {
+typedef struct {
     bool isStopping;
```

Comment:
Karatebrot explicitly requested that I add `earthFrame_s` as the name here.  IDK who to follow.